src/FOHInterface/FOHController.java

```diff
@@ -36,19 +37,26 @@ public int getAmountBooked(long timestamp) {
 
     @Override
     public void setServiceCharge(int newCharge) {
+
         Bill.setServiceChargePercentage(newCharge);
+        String notification = "The service charge now is: " + newCharge +"%";
+        Notification.addNotification(notification);
     }
 
     @Override
     public void markOrderComplete(int orderID) {
         OrderCollection.get(orderID).setComplete(true);
         OrderDAO.setOrderStatusToReady(orderID);
+        String notification = "The Order: " +  OrderCollection.get(orderID).getOrderId() + " is ready to collect.";
+        Notification.addNotification(notification);
     }
 
     @Override
     public void markItemUnavailable(int itemID) {
         DishDAO.setDishUnavailable(itemID);
         MenuItemCollection.get(itemID).setAvailable(false);
+        String notification = "The dish: " +  MenuItemCollection.get(itemID).getName() + " is unavailable.";
```

Comment:
This is good, but I'd say send the name as well as the itemID

Reply:
sure